README.md

```diff
@@ -2,14 +2,21 @@
 
 [![Build Status](https://travis-ci.org/axa-ch/patterns-library.svg?branch=develop)](https://travis-ci.org/axa-ch/patterns-library)
 
-[Pattern Library Preview](https://patterns.axa.ch)
+## [>> Pattern Library](https://patterns.axa.ch)
```

Comment:
Please leave preview as it is a preview

Reply:
People will not realise is a link to the preview

Reply:
I find that "Peview" is an unfortunate name. It's basically saying: "Well, we are not ready yet, but here you can take a glimpse at the current progress." That's not actually the case, since we have components released that can be used. Therefore we shouldn't put the prototype label on ourselves. Otherwise people won't take us seriously when we tell them that we are ready and on and then put a preview label. Can't we find another word for out "Storybook" presentation? Because I didn't know any better, that's why I removed it. Something like "Pattern Library Live" or whatever. I don't like that one too, but just as example. :)

Reply:
Well why change the fact. It is a preview of our components. The client does not use our components from storybook but from npm and storybook is a pre-view of what he gets from npm. It doesn’t mean at all that they are not finished. Pre view mean before viewing so i dont think there is a better matching name. Live is less specific. Please leave preview

Reply:
![C9BBDA22-98E9-445E-BD22-6C0A66A72A72](https://user-images.githubusercontent.com/275235/58305889-e98a6280-7df9-11e9-8299-473ed8cc320d.png)

Reply:
But if u cant live with preview use sometimes like: live, url, link, visible etc